MicrosoftTeams.schema.json

```diff
@@ -369,79 +369,16 @@
-                                "initialRun": {
-                                    "type": "boolean",
-                                    "description": "A boolean value that indicates if the command should be run once initially with no parameter.",
-                                    "default": false
+                                {
+                                    "$ref": "#/definitions/actionFetchCommand"
                                 },
-                                "fetchTask": {
-                                    "type": "boolean",
-                                    "description": "A boolean value that indicates if it should fetch task module dynamically",
-                                    "default": false
-                                },
-                                "parameters": {
-                                    "type": "array",
-                                    "maxItems": 5,
-                                    "minItems": 1,
-                                    "items": {
-                                        "type": "object",
-                                        "additionalProperties": false,
-                                        "properties": {
-                                            "name": {
-                                                "type": "string",
-                                                "description": "Name of the parameter.",
-                                                "maxLength": 64
-                                            },
-                                            "inputType": {
-                                                "type": "string",
-                                                "enum": [ "text", "textarea", "number", "date", "time", "toggle" ],
-                                                "description": "Type of the parameter"
-                                            },
-                                            "title": {
-                                                "type": "string",
-                                                "description": "Title of the parameter.",
-                                                "maxLength": 32
-                                            },
-                                            "description": {
-                                                "type": "string",
-                                                "description": "Description of the parameter.",
-                                                "maxLength": 128
-                                            }
-                                        },
-                                        "required": [
-                                            "name",
-                                            "title"
-                                        ]
-                                    }
+                                {
+                                    "$ref": "#/definitions/actionParamCommand"
```

Comment:
Nit pick, but shouldn't just be commandParam or something? Cause technically params are associated with query as well